parcels/field.py

```diff
@@ -286,27 +270,19 @@ def _get_dim_filenames(cls, filenames, dim):
             return filenames
 
     @staticmethod
-    def _collect_timeslices(data_filenames, dimensions, indices):
-        timeslices = []
-        dataFiles = []
+    def _collect_time(data_filenames, dimensions, indices):
+        time = []
         for fname in data_filenames:
             with NetcdfFileBuffer(fname, dimensions, indices) as filebuffer:
                 ftime = filebuffer.time
-                timeslices.append(ftime)
-                dataFiles.append([fname] * len(ftime))
-        time = np.concatenate(timeslices).ravel()
-        dataFiles = np.concatenate(dataFiles).ravel()
+                time.append(ftime)
+        time = np.concatenate(time).ravel()
         if time.size == 1 and time[0] is None:
             time[0] = 0
         time_origin = TimeConverter(time[0])
         time = time_origin.reltime(time)
 
-        if not np.all((time[1:] - time[:-1]) > 0):
-            id_not_ordered = np.where(time[1:] < time[:-1])[0][0]
-            raise AssertionError(
-                f"Please make sure your netCDF files are ordered in time. First pair of non-ordered files: {dataFiles[id_not_ordered]}, {dataFiles[id_not_ordered + 1]}"
-            )
```

Comment:
Is this not an issue anymore? Or will this part of the Field code be removed later in v4 dev anyways?

Reply:
I think that this is handled natively in `xr.open_mfdataset()`